lib/i18n-base.types.ts

```diff
@@ -1,3 +1,7 @@
 export interface Translation {
   [key: string]: string;
 }
+
+export interface AllTranslations<T> {
```

Comment:
i would rather like this here than the one in the template, but then we need to access this path from the generated file...which should not be too difficult, if we use the node_modules path?

Reply:
totally agree!